### PR TITLE
PLF-8673:get the real name of uploaded file(Activity Stream)

### PR DESCRIPTION
**Pull request by sbouhlel**

### CODE REVIEW

File: integ-ecms/integ-ecms-social/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivity.java [hunk 1]
```diff
@@ -1302,7 +1302,10 @@ private String getDownloadURL(DownloadService dservice,
       }
 
       ActivityFilesDownloadResource dresource = new ActivityFilesDownloadResource(nodelocations);
-      dresource.setDownloadName(fileName);
+      //we should decode twice to get the real name(File with special character)
+      String fileName1=URLDecoder.decode(fileName,"UTF-8");
```
Comment (ahamdi) [inline integ-ecms/integ-ecms-social/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivity.java]: Do it in one line without the need of two extra variables fileName1 and fileName2

Comment (sbouhlel) [reply integ-ecms/integ-ecms-social/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivity.java]: Done

File: integ-ecms/integ-ecms-social/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivity.java [hunk 2]
```diff
@@ -252,7 +252,7 @@ public String getContentName(int i) {
       if (StringUtils.isBlank(contentName)) {
         contentName = URLDecoder.decode(activityFileAttachment.getContentName(), "UTF-8");
       }
-      return URLDecoder.decode(contentName, "UTF-8");
+      return URLDecoder.decode(activityFileAttachment.getContentNode().getProperty("exo:title").getString(), "UTF-8");
```
Comment (hbenali) [inline integ-ecms/integ-ecms-social/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivity.java]: I see the contentName variable is no longer used. Why you kept it?

**Comment by boubaker** [comment]
The project 'platform' has been deleted for 6.0.x, thus please move the content of this PR to 'ecms'

**Pull request closed without merging**